### PR TITLE
HARP-5995: Fix free camera example.

### PR DESCRIPTION
**Pull request by kkostecki**
This includes:
- displaying camera frustum,
- rendering camera transformation gizmo,
- adding support for camera angles constraints.

Signed-off-by: Krystian Kostecki <ext-krystian.kostecki@here.com>

Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

**Comment by kkostecki** [comment]
@heremaps-bot ok to test

**Comment by Astrak** [comment]
@heremaps-bot retest this please

**Comment by kkostecki** [comment]
@heremaps-bot retest this please

**Comment by kkostecki** [comment]
@heremaps-bot retest this please

File: @here/harp-examples/src/getting-started_free-camera.ts [hunk 1]
```diff
@@ -139,20 +146,54 @@ export namespace FreeCameraAppDebuggingToolExample {
...
+
+            const applyTransformControls = () => {
+                // Apply helper camera offset to main (map view) camera.
+                this.mapView.camera.position.add(cameraRelativeToEye.position);
+                // Make sure that pitch limit contraint is preserved
+                const ypr = MapViewUtils.extractYawPitchRoll(
+                    cameraRelativeToEye.quaternion,
+                    this.mapView.projection.type
+                );
+                ypr.pitch = Math.max(
```
Comment (zbigg) [inline @here/harp-examples/src/getting-started_free-camera.ts]: For future: we've got `MathUtills.clamp` in `@here/harp-utils`.

Comment (kkostecki) [reply @here/harp-examples/src/getting-started_free-camera.ts]: Yeah, I have used it even :).

**Pull request merged**